docker/Dockerfile.bionic-static

```diff
@@ -0,0 +1,56 @@
+FROM ubuntu:bionic as bccbuilder
```

Comment:
I don't see the `bccbuilder` alias used anywhere else

Reply:
Yeah this was an artifact from an earlier attempt to only copy the necessary BCC libs and headers. You're correct it is no longer necessary.